docs/modules/administration/pages/update-2-to-3.adoc

```diff
@@ -0,0 +1,137 @@
+== Update your OpenDevStack repositories
+
+Updating repositories means that new refs from repositories under
+`github.com/opendevstack` are pushed into the repositories in your BitBucket
+instance.
+
+First, you need a clone of each repository in BitBucket which should be updated
+on your local machine.
+
+Once this has been done, you need to fetch new refs from
+`github.com/opendevstack`. To do so, add a remote pointing to it like this:
+
+[source,sh]
+----
+git remote add ods https://github.com/opendevstack/<REPO_NAME>.git
+----
+
+Now you are ready to update the refs. It is recommended to update both the
+`master` branch and, unless you want to live off the bleeding edge, a release
+branch such as `2.x`. Use the steps shown below:
+
+[source,sh]
+----
+# Ensure you have the latest refs from ODS locally
+git fetch ods
+# Update master
+git checkout master
+git reset --hard ods/master
+git push origin master
+# Update 2.x
+git checkout 2.x
+git reset --hard ods/2.x
+git push origin 2.x
+----
+
+If your OpenDevStack installation is based on a custom branch (such as `2.acme`), then you
+need to create a pull request on BitBucket from `2.x` into that custom branch now.
```

Comment:
I think pretty much all of the above can be replaced by `make sync-repos` (which asks which revision to use).

For custom branch, people need to run sth like `ods-setup/repos.sh --source-git-ref master --target-git-ref 3.x`